docs/src/components/LeftNav/LeftNav.js

```diff
@@ -60,7 +60,7 @@ const LeftNav = (props) => {
   };
 
   return (
-    <div className='h-100 bg-secondary-lightest border-right'>
+    <div className='h-100 bg-secondary-lightest border-right' style={{overflow: "scroll"}}>
       {showMenuButtons && (
```

Comment:
- Instead of a style tag please use className
- We don't require an overflow-x in this case. Only add overflow-y
- Use scrollbar similar to Design system

Reply:
done